.github/workflows/node-ci.yml

```diff
@@ -118,7 +118,7 @@ jobs:
       uses: preactjs/compressed-size-action@v2
       with:
         # We only care about the browser ES module size, really:
-        pattern: "**/dist/es/index.browser.js"
+        pattern: "**/dist/browser/es/index.js"
```

Comment:
**note:** This change might mean the size check fails for this PR, but it'll start working again once it is landed.